orchestrator/test_runner/src/transaction_stress_test.rs

```diff
@@ -107,22 +106,27 @@ pub async fn transaction_stress_test(
                 if !found {
                     good = false;
                 }
+                if good {
+                    break;
+                }
             }
+            tokio::time::sleep(std::time::Duration::from_secs(5)).await;
```

Comment:
If we don't have an enclosing outer loop within the `timeout` call, there's no reason to delay. I think the original intent was that it would try multiple times until it succeeded or hit the `TOTAL_TIMEOUT`. Just using the `for` loop means it tries once, sleeps for a bit, and then returns. As in the other examples, probably want to add an overall outer `loop`.

Reply:
This commit: https://github.com/PeggyJV/gravity-bridge/pull/274/commits/4d7cc8f741f554794da2abf6e8eb4feb9bf9cf78 fixes this. I'm resolving this comment now